rkt/images.go

```diff
@@ -402,6 +405,26 @@ func (f *fetcher) fetch(aciURL, ascURL string, ascFile *os.File) (*openpgp.Entit
 		defer ascFile.Close()
 	}
 
+	// check if the identity used by the signature is in the store before a
+	// possibly expensive download. This is only an optimization and it's
+	// ok to skip the test if the signature will downloaded later.
+	if !retrySignature && f.ks != nil && appName != "" {
+		emptyFile, err := ioutil.TempFile("", "rkt-empty-file-")
```

Comment:
huh? this is what interfaces are for :-) - just pass an empty bytes.Buffer into CheckSignature